README.md

```diff
@@ -0,0 +1,190 @@
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Portfolio] <a name="about-project"></a>
+
+
+
+**[Portfolio]** is a Microverse Project bootcamp to learn about Mobile first and responsive desing.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+- HTML
+- CSS
+- JavaScript
+- GitHub
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+
+- **[Mobile Designer]**
+- **[Headline]**
+- **[Tourism page]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+[Click Here](https://ulises2607.github.io/cayafate-project/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+
+- Any internet browser.
+- Git or Github for desktop.
+
+### Setup
+
+Clone this repository to your desired folder:
+
+
+```sh
+  cd my-folder
+  git@github.com:ulises2607/cayafate-project.git
+```
+
+
+### Install
+
+Install this project with:
+
+- N/A
+
+### Usage
+
+-N/A
+
+### Run tests
+
+To run tests, run the following command:
+
+```sh
+  npx hint .
+```
+```sh
+  npx stylelint "**/*.{css,scss}"
+```
+```sh
+  npx eslint .  
+```
+
+### Deployment
+
+#### If you want to deploy the website you can use the free webportals or platforms for deloyment. Like `gh-pages`, `vercel`, `netlify`and so many other free platforms. This website is deploye using `gh-pages`.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+
+👤 **Author1**
+
+- GitHub: [@ulises2607](https://github.com/ulises2607)
+- LinkedIn: [Cesar Ulises Cappa Subelza](https://www.linkedin.com/in/cesar-ulises-cappa-subelza/)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+
+- [1] **[Popups]**
+- [2] **[Animations]**
+- [3] **[Images cards]**
+
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+- Microverse Code Reviewers
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project please comment on what you liked about it, and what I can improve on to make it better.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+
+I would like to thank microverse and my code reviewers team for provide me the tools to make this project
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- [ ] Please 🙏🏻 adhere to the project requirements properly according to which you should have to give the credit to the designer **Cindy Shin**, which shows that you are following the design usage terms and conditions. A demo video is attached below.

[Screencast from 08-07-23 04:11:15 PM IST.webm](https://github.com/ulises2607/cayafate-project/assets/66476329/92684188-88cf-4dd7-98a6-f14780f05722)